nifi-nar-bundles/nifi-hazelcast-bundle/nifi-hazelcast-services/src/main/resources/org.apache.nifi.hazelcast.services.cachemanager.EmbeddedHazelcastCacheManager/additionalDetails.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>EmbeddedHazelcastCacheManager</title>
+    <link rel="stylesheet" href="/nifi-docs/css/component-usage.css" type="text/css" />
+</head>
+
+<body>
+<h2>EmbeddedHazelcastCacheManager</h2>
+
+<p>
+    This service starts and manages an embedded Hazelcast instance. The instance needs to listen on a port for
+    possible clients to join in. The cache manager will not join via this port but directly reaches data using the embedded
+    server instance. This behaviour comes from Hazelcast and not to be prevented but not used as the embedded manager
+    is intended to be serve as an internal cache storage. However, be aware: this might make the instance reachable
+    within the same network and it is possible for clients to join.
```

Comment:
```suggestion
    This service starts and manages an embedded Hazelcast instance. The cache manager has accesses to the instance - and the data stored in it - directly. However, the instance sill opens a port for potential clients to join and this cannot be prevented. Note that this might leave the instance open for rogue clients to join.
```